Gruntfile.js

```diff
@@ -199,8 +199,8 @@ module.exports = function (grunt) {
           files: ['Gruntfile.js'],
           tasks: ['eslint']
         },
-        js: {
-          files: ['Gruntfile.js', 'src/**/*.js'],
+        all: {
+          files: ['Gruntfile.js', 'src/**/*.js', 'src/**/*.html', 'styles/**/*.css'],
           tasks: ['build']
```

Comment:
I feel like this is a simple, non-intrusive, change and doesn't really warrant a separate PR ;-)